.laminas-ci/composer-root-version.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -l
+
+if [[ -z "${GITHUB_BASE_REF}" ]]; then
+    echo "Environment variable \"GITHUB_BASE_REF\" does not exist."
+
+    exit 0
+fi
+
+BRANCH_REGEX="[0-9]+\.[0-9]+\.x"
+
+if ! [[ "${GITHUB_BASE_REF}" =~ ${BRANCH_REGEX} ]]; then
+    echo "Environment variable \"GITHUB_BASE_REF\" does not match expectations."
+    echo "Must match ${BRANCH_REGEX}";
+
+    exit 0
+fi
+
+COMPOSER_ROOT_VERSION=$(echo ${GITHUB_BASE_REF} | sed 's/\.x/\.99/g')
```

Comment:
Do you think there's a way to centralize this file for all satellite components instead of copy-paste it everywhere?
Not a big issue tough, just a little brain-storming

Reply:
I think its not possible.
The file has to be available after checkout and there is no composer package installed when this is executed.

I think, for the time being, we have to copy and paste it. I wanted this to be part of the action itself but I forgot why this was not wanted.

----

When creating v2.0 of the satellites targeting v3.0 of laminas-cache, the circular dependency is dropped and we can delete that file.